src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <h1 class="heading">HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="input-wrapper">
+          Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+          >
+        </label>
+        <label class="input-wrapper">
+          Name:
+          <input
+           type="text"
+           name="name"
+           autocomplete="off"
+          >
+        </label>
+        <label class="input-wrapper">
+          How old are You?
+          <input
+            type="number"
+             name="age"
+             min="1"
+             max="100"
+             value="12"
+          >
+        </label>
+        <label class="input-wrapper">
+          Full date of birth:
+          <input type="date" name="birthDate">
+        </label>
+        <label class="input-wrapper">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            id="terms"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="input-wrapper">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label class="input-wrapper">
+          Password:
+          <input type="password" name="password">
```

Comment:
Suggestion:
It seems to me it is easier to read this way
```suggestion
          <input 
            type="password" 
            name="password"
          >
```